src/diffusers/pipelines/deepfloyd_if/watermark.py

```diff
@@ -0,0 +1,44 @@
+from typing import List
+
+import PIL
+import torch
+from PIL import Image
+
+from ...configuration_utils import ConfigMixin
+from ...models.modeling_utils import ModelMixin
+from ...utils import PIL_INTERPOLATION
+
+
+class IFWatermarker(ModelMixin, ConfigMixin):
+    def __init__(self):
+        super().__init__()
+
+        self.register_buffer("watermark_image", torch.zeros((62, 62, 4)))
+        self.watermark_image_as_pil = None
+
+    def apply_watermark(self, images: List[PIL.Image.Image], sample_size):
```

Comment:
```suggestion
    def forward(self, images: List[PIL.Image.Image], sample_size):
```

Reply:
And apply to all pipelines - @williamberman could you take care of this?

Reply:
discussed on slack